api/types/types.go

```diff
@@ -155,3 +155,27 @@ type Info struct {
 	Name               string
 	Labels             []string
 }
+
+type ContainerState struct {
```

Comment:
As long as it matches what is sent on the wire.  As I mentioned in irc though, I'd want the full json represented, not just the bits the CLI uses.